src/cow-react/modules/trade/pure/TradeWidgetField/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Wrapper, Label, Content, ErrorText, WarningText } from './styled'
+import { Trans } from '@lingui/macro'
+import { Nullish } from '@cow/types'
+import QuestionHelper from 'components/QuestionHelper'
+
+export interface TradeWidgetFieldProps {
+  label: string
+  children?: JSX.Element
+  hint?: JSX.Element | string
+  error?: Nullish<string>
```

Comment:
Maybe better to do smth like in `TradeWarning`?


```
type TradeWidgetFieldError = { type: 'error' | 'warning', text: string }

export interface TradeWidgetFieldProps {
    error: TradeWidgetFieldError
}
```

Following this pattern, we can easily extend types of `TradeWidgetFieldError['type']`.
Also, we can rid of this ambiguity:

```
{error && <ErrorText>{error}</ErrorText>}
{warning && <WarningText>{warning}</WarningText>}
``

Now it looks like we can have displayed error and warning at the same time.

Reply:
Changed this